config/default/forms/app/geopoint.xml

```diff
@@ -0,0 +1,24 @@
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
```

Comment:
It think this is a little mishap, I don't think we should be adding the geopoint demo form to the CHT reference app.

Reply:
@dianabarsan  - do you mean we should remove it from the `demo-forms` [directory](https://github.com/medic/cht-core/tree/master/demo-forms)? I don't think this form is otherwise included in [the default app](https://github.com/medic/cht-core/tree/master/config/default).  I should confess I'm not entirely sure if CHT reference app == [default app](https://github.com/medic/cht-core/tree/master/config/default), so please correct me if I'm wrong!

Reply:
Fixed in 0e7ca0c  - thanks again!